src/core.c/Exception.pm6

```diff
@@ -351,31 +351,36 @@ my class X::Pragma::OnlyOne is Exception {
 
 my role X::Control is Exception {
 }
-my class CX::Next does X::Control {
+my role CX::Returning {
```

Comment:
Can we find a better name for the role? To my view neither `take` nor `emit` are not a return-kind things.

Reply:
Sure. What is your suggestion?

Reply:
Thinking about it a little longer I would like to point the following out. If I borrow your lawn mower and give it to your wife after using it, did I return the lawn mower? Do I have to put it back into your shed or will the garage do? Let's have a look of the common usage we have for the english term "return" in Raku.

```
sub borrow($thing) {
    return "short sharp knife";
}

say borrow("lawn mower");
```

I strongly doubt many functions are actually "returning" anything. Some functions even have a completely empty signature and yet, they "return" a value that was never given to them. I'm quite sure we can come up with a differently wrong english phrase. Not sure if that would be better.

Reply:
To my mind, what `return` returns (and the returner utilized on behalf of the returnee) is *control over the process*. Maybe structured programming is so natural today that this meaning doesn't stand out anymore.  Raku has a number of different words for this same concept, which are used depending on who transfers control (a `sub` call, a `supply` block, a `when` block, a `gather` block, ...). In fact, this meaning of "returning" could be said to apply to all control exceptions, save for CX::Warn, which is opposite of vrurg's concern. The new CX::Returning applies to control exceptions which *carry a value* to the returnee, but in that direction all I can come up with are `Valuable` / `Valued` / `Payloaded` neither of which sounds good.

Reply:
> Sure. What is your suggestion?

Unfortunately, my English fails me here. In a personal project I'd name it `Payloadish`, but not sure if everybody would like it here. 

Alternatively, my brain have came up with `CX::Carrier` variant a few minutes ago.

Reply:
How about `class CX::SomeNameHere does Payload {...}` ?

Reply:
> How about `class CX::SomeNameHere does CX::Payload {...}` ?

Look undoubtedly better than `CX::Returning`. I'd stop here.

I still like `CX::Carrier` anyway... ;)